### PR TITLE
Ambassadors frontend

### PR DESCRIPTION
**Pull request by philamwild**

### CODE REVIEW

**Review by BastienSaulnier — DISMISSED** [review]
SnE

File: frontend/src/pages/AmbassadorInfos.jsx [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+function AmbassadorInfos(props) {
+  const [ambassador, setAmbassador] = useState([]);
+
+  useEffect(() => {
+    axios
+      .get(`http://localhost:4000/ambassador/${props.match.params.id}`)
+      .then(res => {
+        setAmbassador(res.data[0]);
+        console.log(res.data);
```
Comment (LoicBrassart) [inline frontend/src/pages/AmbassadorInfos.jsx]: Ah, no! No `console.log` on my GH !

File: frontend/src/pages/Ambassadors.jsx [hunk 2]
```diff
@@ -1,19 +1,39 @@
...
+
+  useEffect(() => {
+    axios
+      .get(`http://localhost:4000/ambassador`)
+      .then(res => {
+        setAmbassadors(res.data);
+        console.log(res.data);
+      })
+      .catch(err => {
+        console.log(err);
```
Comment (LoicBrassart) [inline frontend/src/pages/Ambassadors.jsx]: Same!

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
Don't leave logs in the production code

**Review by BastienSaulnier — DISMISSED** [review]
SnE anymore, did changes request

File: frontend/src/pages/AmbassadorInfos.jsx (hunk 1 above)
Comment (BastienSaulnier) [reply frontend/src/pages/AmbassadorInfos.jsx]: Done

File: frontend/src/pages/Ambassadors.jsx (hunk 2 above)
Comment (BastienSaulnier) [reply frontend/src/pages/Ambassadors.jsx]: Done

File: frontend/src/pages/Home.jsx [hunk 3]
```diff
@@ -15,7 +16,7 @@ function Home() {
 
   const getCampaignInfo = () => {
     axios
-      .get("http://localhost:4000/campaign/")
+      .get(`${apiCall}/campaign/`)
       .then(res => setCampaignInfo(res.data))
       .catch(err => console.log(err));
```
Comment (LoicBrassart) [inline frontend/src/pages/Home.jsx]: No console.log

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
SnE, except for a wild `console.log`
Fiix this and it's ok

**Review by LeoDiduch — APPROVED** [review]
SnE

File: frontend/src/pages/Home.jsx (hunk 3 above)
Comment (BastienSaulnier) [reply frontend/src/pages/Home.jsx]: Done

**Review by BastienSaulnier — APPROVED** [review]
Sne ANYMORE

**Review by LoicBrassart — APPROVED** [review]
SnE

**Pull request merged**